playbooks/awx.yml

```diff
@@ -9,3 +9,11 @@
   roles:
     - role: ../roles/pulibrary.nodejs
     - role: ../roles/pulibrary.awx
+
+  post_tasks:
+    - name: tell everyone on slack you ran an ansible playbook
+      slack:
```

Comment:
Are we going to need to specify a required minimum release for Ansible in the README? https://docs.ansible.com/ansible/latest/modules/slack_module.html states that this is currently a part of core.

Reply:
Our contributions doc `assumes` you are always using the latest stable (I know it isn't always our practice) 
https://github.com/pulibrary/princeton_ansible/blob/master/CONTRIBUTING.md

Reply:
ping @jrgriffiniii

Reply:
Sorry, I had overlooked that.  I am still 👍then (that wasn't meant to block approval).